static/css/latex.css

```diff
@@ -291,15 +429,6 @@ dl dd {
   border-top: 1px solid hsl(0, 0%, 39%);
 }
 
-/* Center title and paragraph */
```

Comment:
I would prefer to set the "abstract" center-aligned, and, since LateX.css also do so, I'd like to make it the default.
(Of course you can modify it in your own project!)